package.json

```diff
@@ -32,7 +32,6 @@
     "react-paginate": "^5.0.0",
     "react-redux": "^5.0.6",
     "react-transition-group": "^2.2.1",
-    "reactstrap": "^4.8.0",
```

Comment:
I think this was left here by mistake at some point. We don't use reactstrap anywhere in SFE.